kerascv/layers/anchor_generators/anchor_generator.py

```diff
@@ -1,7 +1,47 @@
+# Copyright 2020 The Keras CV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 import tensorflow as tf
 
 
 class AnchorGenerator(tf.keras.layers.Layer):
+    """Defines a AnchorGenerator that generates anchor boxes for a single feature map.
+
+    # Attributes:
+        image_size: A list/tuple of 2 ints, the 1st represents the image height, the 2nd image width.
+        scales: A list/tuple of positive floats (usually less than 1.) as a fraction to shorter side of `image_size`.
+            It represents the base anchor size (when aspect ratio is 1.). For example, if `image_size` is (300, 200),
+            and `scales=[.1]`, then the base anchor size is 20.
+        aspect_ratios: a list/tuple of positive floats representing the ratio of anchor width to anchor height.
+            **Must** have the same length as `scales`. For example, if `image_size=(300, 200)`, `scales=[.1]`,
+            and `aspect_ratios=[.64]`, the base anchor size is 20, then anchor height is 25 and anchor width is 16.
+            The anchor aspect ratio is independent to the original aspect ratio of image size.
+        anchor_stride: A list/tuple of 2 ints or floats representing the distance between anchor points.
+            For example, `anchor_stride=(30, 40)` means each anchor is separated by 30 pixels in height, and
+            40 pixels in width. Defaults to `None`, where anchor stride would be calculated as
+            `min(image_height, image_width) / feature_map_height` and
+            `min(image_height, image_width) / feature_map_width`.
+        anchor_offset: A list/tuple of 2 floats between [0., 1.] representing the center of anchor points relative to
+            the upper-left border of each feature map cell. Defaults to `None`, which is the center of each
+            feature map cell when `anchor_stride=None`, or center of anchor stride otherwise.
+        clip_boxes: Boolean to represents whether the anchor coordinates should be clipped to the image size.
+            Defaults to `True`.
+        norm_coords: Boolean to represents whether the anchor coordinates should be normalized to [0., 1.]
```

Comment:
In general, prefer fully-spelled out argument names, e.g. `normalize_coordinates`

Reply:
Done.